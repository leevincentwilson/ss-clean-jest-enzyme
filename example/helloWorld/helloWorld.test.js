import { configure, shallow } from 'enzyme';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import HelloWorld from './index';

configure({ adapter: new Adapter() });

describe('<HelloWorld />', () => {
  test('It should render the hello world component', () => {
    const wrapper = shallow(<HelloWorld />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
