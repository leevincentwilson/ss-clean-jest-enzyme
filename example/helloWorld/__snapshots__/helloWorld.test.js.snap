// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloWorld /> It should render the hello world component 1`] = `
<div>
  Hello World!
</div>
`;
